models/prices/zora/prices_zora_tokens.sql

```diff
@@ -0,0 +1,19 @@
+{{ config(
+        schema='prices_zora',
+        alias = 'tokens',
+        materialized='table',
+        file_format = 'delta',
+        tags = ['static']
+        )
+}}
+SELECT
+    token_id
+    , blockchain
+    , symbol
+    , contract_address
+    , decimals
+FROM
+(
+    VALUES
+    ('weth-weth', 'zora', 'WETH', 0x4200000000000000000000000000000000000006, 18)
```

Comment:
I will try to add weth without modify to `scripts/token_checker.py` and see what will happen.